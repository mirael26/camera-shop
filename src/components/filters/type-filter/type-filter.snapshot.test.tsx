import { renderWithReduxAndRouter } from '../../../test/helpers/render-with-redux-and-router';
import TypeFilter from './type-filter';

test('TypeFilter component renders correctly', () => {
  const {asFragment} = renderWithReduxAndRouter(<TypeFilter/>);
  expect(asFragment()).toMatchSnapshot();
});
