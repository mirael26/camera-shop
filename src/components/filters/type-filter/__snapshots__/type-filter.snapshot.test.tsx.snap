// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TypeFilter component renders correctly 1`] = `
<DocumentFragment>
  <fieldset
    class="catalog-filter__block"
  >
    <legend
      class="title title--h5"
    >
      Тип камеры
    </legend>
    <div
      class="custom-checkbox catalog-filter__item"
    >
      <label>
        <input
          name="Цифровая"
          type="checkbox"
        />
        <span
          class="custom-checkbox__icon"
        />
        <span
          class="custom-checkbox__label"
        >
          Цифровая
        </span>
      </label>
    </div>
    <div
      class="custom-checkbox catalog-filter__item"
    >
      <label>
        <input
          name="Плёночная"
          type="checkbox"
        />
        <span
          class="custom-checkbox__icon"
        />
        <span
          class="custom-checkbox__label"
        >
          Плёночная
        </span>
      </label>
    </div>
    <div
      class="custom-checkbox catalog-filter__item"
    >
      <label>
        <input
          name="Моментальная"
          type="checkbox"
        />
        <span
          class="custom-checkbox__icon"
        />
        <span
          class="custom-checkbox__label"
        >
          Моментальная
        </span>
      </label>
    </div>
    <div
      class="custom-checkbox catalog-filter__item"
    >
      <label>
        <input
          name="Коллекционная"
          type="checkbox"
        />
        <span
          class="custom-checkbox__icon"
        />
        <span
          class="custom-checkbox__label"
        >
          Коллекционная
        </span>
      </label>
    </div>
  </fieldset>
</DocumentFragment>
`;
